en/docs/install-and-setup/setup/api-controller/managing-apis-in-kubernetes/managing-apis-in-kubernetes.md

```diff
@@ -0,0 +1,168 @@
+# Managing APIs in Kubernetes
+
+WSO2 API Controller, **apictl** allows you to manage APIs in Kubernetes with the use of WSO2 API operator. This approach uses the kubernetes native way to deploy the APIS into a running microgateway cluster from the command line.
```

Comment:
```suggestion
**WSO2 API Controller (apictl)** allows you to manage APIs in Kubernetes with the use of WSO2 API Operator. This approach uses the Kubernetes native way to deploy the APIS into a running microgateway cluster from the command line.
```
Recently we decided to introduce **apictl** abbreviation as shown above when starting any apictl related doc. After that, you can use this abbreviation through the doc. We can do the same for API Operator as well if there is an abbreviation for that.

Reply:
Fixed with 21f466ce5d99225a24a45351eae0997e87f8c2f5